reconcile/terraform_resources.py

```diff
@@ -260,8 +260,16 @@ def setup(print_only, thread_pool_size, internal, use_jump_host,
     namespaces = gqlapi.query(TF_NAMESPACES_QUERY)['namespaces']
     tf_namespaces = [namespace_info for namespace_info in namespaces
                      if namespace_info.get('managedTerraformResources')]
-    ri, oc_map = fetch_current_state(tf_namespaces, thread_pool_size,
-                                     internal, use_jump_host)
+    if dry_run:
+        ri = ResourceInventory()
```

Comment:
The `fetch_current_state` should be the function with the knowledge about what to give you in case of `dry_run`. Can you make it accept the `dry_run` argument and return this?

Reply:
that makes a lot of sense! thanks!